lib/absinthe/phase/document/complexity/analysis.ex

```diff
@@ -36,6 +36,7 @@ defmodule Absinthe.Phase.Document.Complexity.Analysis do
     end)
   end
 
+  def handle_node(%Blueprint.Document.Fragment.Spread{} = node, _info, %{}), do: %{node | complexity: nil}
```

Comment:
This does indeed make the error go away, but it also disables actually doing any complexity analysis on fragment spreads, which would also be a bug.

Reply:
Yes, you are right. what would you advise for this issue? Should `handle_node` view all fragments of the input?